jnigen/lib/src/config/config.dart

```diff
@@ -133,6 +133,13 @@ T _getEnumValueFromString<T>(
   return value;
 }
 
+void _ensureIsDirectory(String name, Uri path) {
+  if (!path.toFilePath().endsWith(Platform.pathSeparator)) {
+    throw ArgumentError('$name must be a directory path. If using YAML '
```

Comment:
Do we need test cases that expect an error to be thrown for the negative cases we're trying to prevent? Possibly also for all other kinds of configs which we give error on. So this probably should be a separate PR then. This would increase our code coverage.

Reply:
Yeah I think error paths aren't checked well in config, binding generators. Last time I checked lcov report for support library I was left with same impression.


Also some checks like pdfbox plugin test aren't covered by test coverage.

There could be separate PR for test coverage.

Ideally should also have test coverage for summarizer and C code in support library. Not sure how to combine all results using coveralls.

Reply:
Let's do it in a separate pr, I've filed https://github.com/dart-lang/jnigen/issues/101.